packages/fscomponents/src/components/serializable/vNext/SerializableImage.tsx

```diff
@@ -14,16 +14,27 @@ export interface SerializableImageProps
     | 'progressiveRenderingEnabled'
     | 'resizeMethod'
     | 'resizeMode'
-    | 'style'
     | 'testID'
   > {
   uri?: string;
-  height?: number;
-  width?: number;
-  style?: ImageStyle;
+  style?: ViewStyle;
+  imageStyle?: ImageStyle;
+  children?: React.ReactNode;
 }
 
```

Comment:
Can we standardize these props? (See SerializableView)
Mainly we are missing the `containerStyle` which is a style that directly wraps the child objects.

Reply:
https://github.com/andyschwob/flagship/blob/43e16a31e845522910dbad0001ace465408fc7c6/packages/fscomponents/src/models/StandardizedComponent.ts#L15-L19